github/files/drupal7/scripts/devops/reinstall/pre_settings.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Adding database variable to settings.php
   sudo: yes
-  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"127.0.0.1\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
+  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"{{ mysql_host }}\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
```

Comment:
Cool, but this patch is not intended to add a secondary or an old database.
The goal is to use a different server by default (as they say there is no time to explain, because I have a reason :)) and I thought that the change in one variable global_settings.yml easier than adding five lines in pre_settings.